lib/bulk_data_test_kit/v1.0.1/bulk_data_smart_backend_services_v1_group.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'smart_app_launch/smart_stu2_suite'
+require_relative 'bulk_data_smart_discovery_v1_group'
+
+module BulkDataTestKit
+  module BulkDataV101
+    class BulkDataSmartBackendServicesV1Group < Inferno::TestGroup
+      title 'SMART Backend Services'
+      id :bulk_data_smart_backend_services_v101
+      optional
+
+      group from: :bulk_data_smart_discovery_v1,
+            config: {
+              inputs: { url: { name: :bulk_server_url } }
+            }
+
+      group from: :backend_services_authorization
```

Comment:
I agree with you!  That's what I originally had hoped to do, but I cannot get it to work in this test kit.  I've tried adding run as group to line 17, before the `group from: :backend_services_authorization`, and well as adding a config setting: `group from: :backend_services_authorization, config { options: run_as_group }`.  

Both result in the discovery tests AND the auth tests being grouped together, which we specifically don't want to do here since the auth tests need to be able to run without the discovery tests to conform to the requirements.  Since the `backend_services_authorization` group is imported from smart app launch, I'm wondering if the `run_as_group` needs to be added to that file in the other test kit, but I'm not always sure how the scope works with grouping and if that would have the intended result (I believe it would require a new release of the smart app launch test kit with the update to show up here as well).  

If you have other ideas let me know!  If not, we may have to let it slide.

Reply:
```suggestion
      group from: :backend_services_authorization,
      run_as_group:true
```

Reply:
Try the suggestion above, I think that will do it!

Reply:
Yes thank you, that worked!  Somehow I never learned that specific syntax but now I know!